core/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -577,8 +577,16 @@ object DeltaTable {
       spark: SparkSession,
       identifier: String,
       partitionSchema: StructType): DeltaTable = {
+    convertToDelta(spark, identifier, partitionSchema, false)
+  }
+
+  def convertToDelta(
```

Comment:
can you also please update https://github.com/delta-io/delta/blob/master/python/delta/tables.py ? we need to keep parity between DeltaTable.scala (spark API) and our PySpark API.